Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -222,11 +289,16 @@ public class HTTPNetworkTransport: NetworkTransport {
                                   error: error)
   }
   
-  private func createRequest<Operation: GraphQLOperation>(for operation: Operation) throws -> URLRequest {
-    let body = requestBody(for: operation)
-    var request = URLRequest(url: self.url)
+  private func createRequest<Operation: GraphQLOperation>(for operation: Operation,
+                                                          httpMethod: GraphQLHTTPMethod,
+                                                          sendQueryDocument: Bool,
+                                                          autoPersistQueries: Bool
+        ) throws -> URLRequest {
```

Comment:
can you pull these onto the same line as the last parameter to match the rest of the code please?

Reply:
updated and other method too.